tests/smoketest/smoketest.sh

```diff
@@ -37,22 +46,23 @@ for NAME in "${CLOUDNAMES[@]}"; do
 done
 
 echo "*** [INFO] Triggering an alertmanager notification..."
-oc run curl --generator=run-pod/v1 --image=radial/busyboxplus:curl -- curl -H "Content-Type: application/json" -d '[{"labels":{"alertname":"Testalert1"}}]' http://alertmanager-operated:9093/api/v1/alerts
+oc run curl --generator=run-pod/v1 --image=quay.io/infrawatch/busyboxplus:curl -- curl -H "Content-Type: application/json" -d '[{"labels":{"alertname":"Testalert1"}}]' http://alertmanager-operated:9093/api/v1/alerts
 # it takes some time to get the alert delivered, continuing with other tests
 
 
 
 # Trying to find a less brittle test than a timeout
-JOB_TIMEOUT=500s
+JOB_TIMEOUT=300s
```

Comment:
On the one hand, shorter timeout is better. On the other, the comment on the line above includes the word "brittle". So I dunno.....

Reply:
300s was the original value, then at some point we made it longer because we thought it wasn't long enough but the reality was that the tripleo containers were not able to be pulled in because of the Dockerhub rate limiting